chapter10/spg/exercise2.pl

```diff
@@ -0,0 +1,34 @@
+#! /usr/bin/perl
+
+use strict;
+use warnings;
+use Benchmark qw(:all);
+
+my @files = glob '*';
```

Comment:
I suppose this evens out the sort performance comparison, although means your functions need to be supplied with these args.